motioneye/motioneye_init.py

```diff
@@ -17,8 +17,10 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import subprocess
+
 import motioneye
 
+
```

Comment:
Just to understand the logic: Why an additional newline was added here? Is it always one empty line to the first variable declaration but two empty lines to the first function definition?

Reply:
Self-answered.